src/algoliasearch.helper.js

```diff
@@ -248,6 +250,55 @@ AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
   });
 };
 
+/**
+ * Start the search for answers with the parameters set in the state.
+ * This method returns a promise.
+ * @return {AlgoliaSearchHelper}
+ * @return {promise.<FacetSearchResult>} the answer results
+ */
+AlgoliaSearchHelper.prototype.searchForAnswers = function(attributesForPrediction, queryLanguages, nbHits) {
```

Comment:
the method needs to be added to the type definition still

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/804/commits/de96b4a734e2d6d5f7c0fb3fcb4d84debf7cb350
not sure how far I should go with the typing.